src/assets/translations/en/main.json

```diff
@@ -2798,7 +2798,36 @@
     "lifetimeRewardsHelper": "The total rewards you have earned from staking your %{symbol} to date",
     "timeStaked": "Time Staked",
     "timeStakedHelper": "The amount of time you have staked your %{symbol}",
-    "stakingReward": "Staking Reward"
+    "stakingReward": "Staking Reward",
+    "claimSelect": {
+      "unavailableClaims": "Unavailable Claims",
```

Comment:
@reallybeard asking for posteriority even though we're probably going to land this PR asap to get things moving: what's the idea re: unavailable claims? Shouldn't we just show what's available and that's about it?